### DISCUSSION TITLE
绕过局域网和大陆模式下，raw.githubusercontent.com 网站不能访问

### DISCUSSION THREAD

**Discussion by m2acgi** [shadowsocks/shadowsocks-android#2981] [Q&A]
Please use the forum at https://discourse.shadowsocks.org/c/implementations/shadowsocks-android instead. Your issue will probably be closed.

例如此图片
https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/WebsiteAssets/images/home-top-img.png

**Comment by yiguihai11** [comment]
盲猜你用的WIFI网络...
不要绕过局域网不然DNS无法被代理(192.168.1.1:53)，选绕过大陆就可以了。

**Comment by m2acgi** [comment]
@yiguihai11 赞！解决了。那么我之前的用法一直是存在 dns 泄露了？另外，它这个内置的域名 ip 分流 acl/规则长期没更新会有影响吗？
